US-CBP/cbp-theme
# Modify tab

#### What's this PR do?
- adds tab functionality to pages
- made tabs dynamic depending on what type of page it is (i.e., component or foundation)
- modify bulma css to darken tab outline

#### Where should the reviewer start?
- `ds-tabs` and `contentHeader` folders

#### How should this be manually tested?

1. build ds-css and ds-ux-guidelines
2. check the tabs on top of the typography and tables pages

#### Any background context you want to provide?
- The color of the tab outline was modified in the `ds-css` folder. 
- I temporarily commented out smooth scrolling in `contentHeader.js` to get hrefs to the different id's working. 

#### What are the relevant tickets?
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/13155212/78085810-0385c180-738a-11ea-90ba-39af3d2a95c1.png)

#### Questions:
- Is there a blog post?
- Does the knowledge base need an update?
